spec/models/latest_blood_pressures_per_patient_per_month_spec.rb

```diff
@@ -5,45 +5,60 @@ def refresh_views
     described_class.refresh
   end
 
+  def refresh_views_with_pg_set_to_reporting_time_zone
+    original = ActiveRecord::Base.connection.execute("SELECT current_setting('TIMEZONE')").first["current_setting"]
+    ActiveRecord::Base.connection.execute("SET LOCAL TIME ZONE '#{Period::REPORTING_TIME_ZONE}'")
+    described_class.refresh
```

Comment:
Ideally this is baked into the `refresh` method. @harimohanraj89 mentioned that when he tried to do that it broke a bunch of specs. Since you might have additional context around this, would you like to take a shot at that @rsanheim? I'm okay with us going ahead with this though.

Reply:
I spent more time reading through various Postgres and Rails timestamp / timezone docs last week, and it still isn't clear to me when ActiveRecord will convert timezones on insert versus when it won't...so any large scale change to our `refresh` method would definitely require some production verification to make sure we are staying consistent and correct.

Reply:
I don't think ActiveRecord itself sets a time zone in postgresql, does it? It would change how it interprets timestamps going in and coming up, but in this case with running a `refresh` there isn't any argument or result set coming out. I think we should be safe.